server/controllers/QuestionController.js

```diff
@@ -112,8 +109,8 @@ export default class QuestionController {
         questionObj,
       });
     } catch (error) {
-      return response.status(500).json({
-        status: 'fail',
+      return response.status(401).json({
```

Comment:
You shouldn't be catching a 401 status error this late in your code. Validations like these should be run before your proper code